packages/recommend/src/createUserAgent.ts

```diff
@@ -0,0 +1,23 @@
+import { UserAgent, UserAgentOptions } from '@algolia/transporter';
+
+// Copied from https://github.com/algolia/algoliasearch-client-javascript/blob/831a3debd0c19c87fd71a3e09396bcb3eacd1f5c/packages/transporter/src/createUserAgent.ts
+// and adapted the initial value for Recommend.
+export function createUserAgent(version: string): UserAgent {
+  const userAgent: UserAgent = {
+    value: `Algolia Recommend for JavaScript (${version})`,
```

Comment:
maybe the default should be:
```suggestion
    value: `Algolia for JavaScript (${version}); Recommend (${version})`,
```

not 100% sure, but this would avoid being marked as an unknown client

Reply:
Agreed, we don't want to differentiate the API client when used for Search vs Recommend. I think we should rather use the `addAlgoliaAgent` to amend the regular default one.

Reply:
Thanks for the rectification. I thought we wanted to differentiate them because they are two different products.

Reply:
See 827022f32f623ce460cb36df593472e5e233d28f.